instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/mongo.go

```diff
@@ -51,10 +52,17 @@ func (m *monitor) Started(ctx context.Context, evt *event.CommandStartedEvent) {
 		semconv.DBSystemMongoDB,
 		semconv.DBOperationKey.String(evt.CommandName),
 		semconv.DBNameKey.String(evt.DatabaseName),
-		semconv.NetPeerNameKey.String(hostname),
 		semconv.NetPeerPortKey.Int(port),
 		semconv.NetTransportTCP,
 	}
+
+	addr := net.ParseIP(hostname)
+	if addr != nil {
+		attrs = append(attrs, semconv.NetPeerIPKey.String(hostname))
```

Comment:
Can you please cover this scenario in the unit tests? It would be good to cover both IPv4 and IPv6.